{{cookiecutter.repo_name}}/README.md

```diff
@@ -0,0 +1,15 @@
+# PythonTemplate
+
+This is an example of a python package / collection of scripts for Nieuwe Warmte Nu teams.
+Use this as a template for starting your project. 
+
+# Copying this template
+
+The template is based on a package called `your_package`. Due to the nature of python and python packaging, this name is spread througout the codebase. 
+Don't forget to change the following: 
+
+* `pyproject.toml` file - This file contains the meta-data for your project. Please check 
+carefully and replace all references to `your_package` with your project name. 
+* `ci/<platform>/*`: several scripts to perform certain tasks. These scripts sometimes reference the package name and need to be changed to kee the script working.  
```

Comment:
The readme was written before the move to cookiecutter,    so this can be removed?

Reply:
Done